extensions/spark/kyuubi-spark-authz/src/main/resources/database_command_spec.json

```diff
@@ -122,7 +137,13 @@
     "catalogDesc" : null,
     "isInput" : false
   } ],
-  "opType" : "CREATEDATABASE"
+  "opType" : "CREATEDATABASE",
+  "uriDescs" : [ {
+    "fieldName" : "path",
+    "fieldExtractor" : "StringURIExtractor",
+    "actionTypeDesc" : null,
```

Comment:
we can remove this actionTypeDesc from desc as it's useless. Let's create a separate PR?

Reply:
DOne